### DISCUSSION TITLE
En Sevilen Şarkıcı

### DISCUSSION THREAD

**Discussion by AcelyaP** [sonsuzus/sonsuzus#329] [Soru Cevap]
Bob'un N şarkı listesi var, her şarkının kendisiyle ilişkilendirilmiş bir şarkıcısı vardır (bir tamsayı ile gösterilir)
Bob'un en sevdiği şarkıcı, şarkıları çalma listesinde en çok olan kişidir.
Bob'un Favori Şarkıcılarının sayısını sayın.

**Giriş Biçimi:**
- İlk satır bir tamsayı N içerir, Bob'un çalma listesindeki şarkıların sayısını belirtir.
- Sıradaki giriş N tamsayıları içerir,  i tamsayısı gösteren şarkıcının i şarkısı.

**Çıkış biçimi:**
Bob'un favori şarkıcılarının sayısı olan tek bir tamsayıyı yazdırın.

**Not:** 64 bit veri türü kullanın

**Kısıtlamalar:**
1≤N≤2*10⁵
1≤a[i]≤10¹⁵

**Comment by sonsuzus** [comment]
Google translate gibi olmuş. Lütfen soruyu düzelt ve anlaşılır hale getir.
